schedule.py

```diff
@@ -469,9 +489,10 @@ def find_all_schedule(model, variables):
     return solutions
 
 
-def main(sheets_url):
+def main(sheets_source):
     # schedule sections and print the result
-    sheets = read_ggsheets(sheets_url)
+    sheets = read_ggsheets(sheets_source)
+    # sheets = read_excel(sheets_source)
```

Comment:
I would default to the URL - unless you want to automatically determine whether it's a URL or not (eg. by checking whether it starts with 'http'